docs/user/guides/setup/git-repo-automatic.rst

```diff
@@ -1,27 +1,54 @@
-How to connect your Git repository
-==================================
+How to automatically configure a Git repository
+===============================================
 
 In this article, we explain how connecting your Read the Docs account to |git_providers_or|
-automatically configures your Git repository and your Read the Docs project.
+makes Read the Docs able to automatically configure your imported Git repositories and your Read the Docs projects.
 
 ✅️ Signed up with your Git provider?
   If you signed up or logged in to Read the Docs with your |git_providers_or|
   credentials, you're all done. Your account is connected.
-  The permissions that are granted are :ref:`explained below <connected-accounts:Permissions for connected accounts>`.
-⬇️ Signed up with your email address?
+
+  The rest of this guide helps to understand how the automatic configuration works.
+
+⏩️️ Signed up with your email address?
   If you have signed up to Read the Docs with your email address,
   you can add the connection to the Git provider afterwards.
-  See: :doc:`/guides/connecting-git-account`
+  You can also add a connection to an additional Git provider this way.
+
+  Please follow :doc:`/guides/connecting-git-account` in this case.
+
+Automatic configuration
+-----------------------
+
+When your Read the Docs account is connected to your Git Provider and you import a Git repository,
+all the basic settings will automatically be configured on the Read the Docs project and on your Git repository.
+
+Here is an outline of what happens:
+
+#. A list of repositories that you have access to are automatically listed on Read the Docs' project import.
+#. You choose a Git repository from the list (see :doc:`/intro/import-guide`).
+#. Data about the repository is now read using the account connection and you are asked to confirm the setup.
+#. Read the Docs creates the project with these basic settings,
+   including a unique webhook that the Git provider calls.
+#. Read the Docs uses the account connection to add its webhook URL on the Git repository's settings,
```

Comment:
Didn't we just mention the webhook in the previous step?

Reply:
Yes, this was _unnecessarily unclear_

The list is now updated and a little easier to understand. But it's possibly much nicer to be able to add diagrams for stuff like this... some other day :)